### PR TITLE
[ES-803]

### PR DESCRIPTION
**Pull request by balaji-alluru**

### CODE REVIEW

File: id-repository/id-repository-core/src/main/java/io/mosip/idrepository/core/config/RedisCacheConfig.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import org.springframework.context.annotation.Configuration;
+import org.springframework.data.redis.cache.RedisCacheConfiguration;
+import org.springframework.data.redis.cache.RedisCacheManager;
+import org.springframework.data.redis.connection.RedisConnectionFactory;
+
+import java.time.Duration;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
```
Comment (LoganathanSekar7627) [inline id-repository/id-repository-core/src/main/java/io/mosip/idrepository/core/config/RedisCacheConfig.java:16]: Add author

File: id-repository/id-repository-core/src/main/java/io/mosip/idrepository/core/config/SimpleCacheConfig.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+import org.springframework.cache.Cache;
+import org.springframework.cache.CacheManager;
+import org.springframework.cache.annotation.CachingConfigurerSupport;
+import org.springframework.cache.concurrent.ConcurrentMapCache;
+import org.springframework.cache.support.SimpleCacheManager;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import com.google.common.cache.CacheBuilder;
+
```
Comment (LoganathanSekar7627) [inline id-repository/id-repository-core/src/main/java/io/mosip/idrepository/core/config/SimpleCacheConfig.java:19]: Add author

File: id-repository/id-repository-core/src/test/resources/application-test.properties [hunk 3]
```diff
@@ -363,6 +363,28 @@ logging.level.org.springframework=OFF
...
+'partner_extractor_formats': 200, \
+'datashare_policies': 200, \
+'topics': 200, \
+'online_Verification_Partners': 200, \
+'uin_encrypt_salt' : 100, \
+'uin_hash_salt': 100 , \
+'id_attributes': 200 }
+
+# Cache expire in seconds is applicable for both 'simple' and 'Redis' cache type
+mosip.esignet.cache.expire-in-seconds={'credential_transaction' : 86400, \
```
Comment (LoganathanSekar7627) [inline id-repository/id-repository-core/src/test/resources/application-test.properties:380]: Where is this property used? can you replace esignet with idrepo?

File: id-repository/id-repository-core/src/test/resources/application-test.properties [hunk 4]
```diff
@@ -363,6 +363,28 @@ logging.level.org.springframework=OFF
 logging.level.root=OFF
 spring.main.banner-mode=off
 
+mosip.idrepo.cache.names=credential_transaction,partner_extractor_formats,datashare_policies,topics,online_Verification_Partners,uin_encrypt_salt,uin_hash_salt,id_attributes
```
Comment (LoganathanSekar7627) [inline id-repository/id-repository-core/src/test/resources/application-test.properties:366]: Where is this used?

**Review by LoganathanSekar7627 — CHANGES_REQUESTED** [review]

File: id-repository/id-repository-core/src/test/resources/application-test.properties (hunk 4 above)
Comment (balaji-alluru) [reply id-repository/id-repository-core/src/test/resources/application-test.properties:366]: This property is being used in simple cache config file sir.

File: id-repository/id-repository-core/src/test/resources/application-test.properties (hunk 3 above)
Comment (balaji-alluru) [reply id-repository/id-repository-core/src/test/resources/application-test.properties:380]: replaced it sir.

File: id-repository/id-repository-core/src/main/java/io/mosip/idrepository/core/config/SimpleCacheConfig.java (hunk 2 above)
Comment (balaji-alluru) [reply id-repository/id-repository-core/src/main/java/io/mosip/idrepository/core/config/SimpleCacheConfig.java:19]: added author details sir.

File: id-repository/id-repository-core/src/main/java/io/mosip/idrepository/core/config/RedisCacheConfig.java (hunk 1 above)
Comment (balaji-alluru) [reply id-repository/id-repository-core/src/main/java/io/mosip/idrepository/core/config/RedisCacheConfig.java:16]: Added author details sir.

**Review by LoganathanSekar7627 — APPROVED** [review]

**Pull request merged**